.ci/install_kata_image_rust.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -o errtrace
+
+cidir=$(dirname "$0")
+rust_agent_repo="github.com/kata-containers/kata-containers"
+osbuilder_repo="github.com/kata-containers/osbuilder"
+
+install_rust() {
+	echo "Installing rust"
+	"${cidir}/install_rust.sh"
+	source $HOME/.cargo/env
+}
+
+build_rust_agent() {
+	go get -d "${rust_agent_repo}" || true
+	pushd "${GOPATH}/src/${rust_agent_repo}/src/agent"
+	rustup target add x86_64-unknown-linux-musl
```

Comment:
To prepare for other architectures, could you change this to use `uname -m` rather than hard-coding `x86_64`? In fact, since this is used below too, I'd create a variable for this rust target toolchain.